LEGO1/lego/legoomni/include/legoracecar.h

```diff
@@ -4,6 +4,39 @@
 #include "legocarraceactor.h"
 #include "legoracemap.h"
 
+struct EdgeReference {
```

Comment:
Please add a `// SIZE` comment and a `DECOMP_SIZE_ASSERT` in the `cpp`. Thank you!